app/models/location.py

```diff
@@ -0,0 +1,26 @@
+from .. import db
+
+
+class Zipcode(db.Model):
+    __tablename__ = 'zip_codes'
+    id = db.Column(db.Integer, primary_key=True)
+    zip_code = db.Column(db.String(10), unique=True)
```

Comment:
@riyer15 let's make it a length 5 string for now and we'll make it a task to possibly expand later and store the original input when we implement parsing.